go/worker/compute/executor/committee/discrepancy.go

```diff
@@ -15,6 +15,18 @@ type discrepancyEvent struct {
 }
 
 func (n *Node) handleDiscrepancy(ctx context.Context, ev *discrepancyEvent) {
+	// TODO: Add round number to discrepancy events (breaking change)
```

Comment:
Not consensus breaking as non-provable events are not part of the events root and are only processed locally. The change is somewhat breaking for downstream tooling that may not expect the new fields, but with 23.0.x the gRPC clients should handle extra fields gracefully.

Reply:
I wasn't aware of that. I have resolved this TODO, so please take a look at this commit, as well as the last two, which have been recently added.